rinex/Cargo.toml

```diff
@@ -35,7 +35,8 @@ geo = { version = "0.23.0", optional = true }
 wkt = { version = "0.10.0", default-features = false, optional = true }
 serde = { version = "1.0", optional = true, default-features = false, features = ["derive"] }
 flate2 = { version = "1.0.24", optional = true, default-features = false, features = ["zlib"] }
-hifitime = { version = "3.7.0", features = ["serde", "std"] }
+hifitime = { version = "3.7.0", features = ["serde", "std", "python"] } 
```

Comment:
@larsnaesbye  
I am not advanced enough into Cargo, but the "python" features for "hifitime" should only be activated if our "pyo3" feature is activated

Reply:
@gwbres
I am also unsure how to do this type of conditional feature activation.